projects/sbb-esta/angular-public/src/lib/select/styles/_select-base.scss

```diff
@@ -3,10 +3,21 @@
 @mixin selectBase {
   .sbb-select-trigger {
     @include inputfields;
+    position: relative;
     cursor: pointer;
     display: flex;
     align-items: center;
-    padding-right: $selectTriggerPaddingRight;
+    padding-right: toPx($selectTriggerPaddingRight);
+
+    @include publicOnly() {
+      @include mq($from: desktop4k) {
+        padding-right: toPx($selectTriggerPaddingRight * $scalingFactor4k);
+      }
+
+      @include mq($from: desktop5k) {
+        padding-right: 92px;
```

Comment:
Usually the desktop5k variant is simply calculated with the $scalingFactor5k. Why is this not the case here?